core/tauri-runtime-wry/src/lib.rs

```diff
@@ -221,11 +221,35 @@ impl<T: UserEvent> Context<T> {
   }
 }
 
+#[derive(Debug, Clone, Default)]
+pub struct ActiveTraceSpanStore(Rc<RefCell<Vec<ActiveTracingSpan>>>);
```

Comment:
What is this used for?

Reply:
to track how much time it took to draw the window, so we keep the span alive for a while